docs/src/analysis/sensitivity.md

```diff
@@ -636,7 +637,7 @@ differentiation as follows:
 function G(p)
   tmp_prob = remake(prob,p=p)
   sol = solve(tmp_prob,Vern9(),abstol=1e-14,reltol=1e-14)
-  res,err = quadgk((t)-> (sum(sol(t)).^2)./2,0.0,10.0,abstol=1e-14,reltol=1e-10)
+  res,err = quadgk((t)-> (sum(sol(t)).^2)/2,0.0,10.0,abstol=1e-14,reltol=1e-10)
```

Comment:
Same here

Reply:
Here it is correct but the ./ is unecessary